images/chaos-daemon/Dockerfile

```diff
@@ -9,10 +9,11 @@ ENV https_proxy $HTTPS_PROXY
 RUN mkdir -p /usr/share/man/man1 /usr/share/man/man2
 
 RUN apt-get update && \
-    apt-get install -y tzdata iptables ipset stress-ng iproute2 fuse util-linux procps curl openjdk-11-jre && \
+    apt-get install -y tzdata iptables ebtables net-tools ipset stress-ng iproute2 fuse util-linux procps curl openjdk-11-jre && \
```

Comment:
could you please explain why require `ebtables` and `net-tools`?

Reply:
The latest version of rs-tproxy need ebtables & ip command.